### ISSUE TITLE
[OpenBMC] rspconfig sshcfg not working

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#3811]
@gurevichmark This doesn't seem to be working quite as expected....

```
[root@briggs01 xcat]# rspconfig node-70e28414280d sshcfg
node-70e28414280d: ssh keys copied to 172.12.253.119
[root@briggs01 xcat]# ssh root@172.12.253.119
root@172.12.253.119's password:
root@xyz123:~# ls -ltr .ssh/authorized_keys
ls: .ssh/authorized_keys: No such file or directory
root@xyz123:~# ls -ltr .ssh
total 0
root@xyz123:~#
```

Can you validate?

**Comment by gurevichmark** [comment]
Fixed by #3817

**Comment by zet809** [comment]
Hi, @whowutwut , can we close this issue if it is closed?
